internal/addrs/output_value.go

```diff
@@ -60,6 +73,68 @@ func (v AbsOutputValue) Equal(o AbsOutputValue) bool {
 	return v.OutputValue == o.OutputValue && v.Module.Equal(o.Module)
 }
 
+func ParseAbsOutputValue(traversal hcl.Traversal) (AbsOutputValue, tfdiags.Diagnostics) {
+	path, remain, diags := parseModuleInstancePrefix(traversal)
+	if diags.HasErrors() {
+		return AbsOutputValue{}, diags
+	}
+
+	if len(remain) != 2 {
+		diags = diags.Append(&hcl.Diagnostic{
+			Severity: hcl.DiagError,
+			Summary:  "Invalid address",
+			Detail:   "An output name is required.",
+			Subject:  traversal.SourceRange().Ptr(),
+		})
+		return AbsOutputValue{}, diags
+	}
+
+	if remain.RootName() != "output" {
+		diags = diags.Append(&hcl.Diagnostic{
+			Severity: hcl.DiagError,
+			Summary:  "Invalid address",
+			Detail:   "Output address must start with \"output.\".",
+			Subject:  traversal.SourceRange().Ptr(),
```

Comment:
I think by this point we've proven that `remain[0]` is valid based on the condition above, so perhaps we could use `remain[0].SourceRange().Ptr()` here instead to refer directly to whatever appears right after the module address, similar to how we call out `remain[1]` in the other error message below. :thinking: 

Admittedly this isn't really a user-facing error message since we will only use absolute output addresses as an implementation detail of our plan format, but ... maybe this ends up serving as the basis for other similar code later and so worth doing some little detail things like this.

Reply:
Good catch, done!